.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,40 @@
-<!-- Please replace #XX below with an existing issue number. Remove the line entirely if none exist. -->
-Fixes #XX
+## Fixes
+<!-- If PR only partly solves an issue, replace 'Fixes' below with 'Partially addresses'. -->
```

Comment:
I'd prefer something less verbose than 'Partially addresses'.  Maybe 'Related to'

Reply:
I like that

Reply:
Reworded in 03feb7f.